src/libraries/Common/src/System/Security/Cryptography/RSAOpenSsl.cs

```diff
@@ -183,7 +182,7 @@ public override byte[] Decrypt(byte[] data, RSAEncryptionPadding padding)
 
             if (destination.Length < rsaSize)
             {
-                Debug.Fail("Caller is responsible for temporary decryption buffer creation");
+                Debug.Fail($"Caller is responsible for temporary decryption buffer creation destination. {nameof(destination)}.{nameof(destination.Length)} == {destination.Length}, {nameof(rsaSize)} = {rsaSize}");
```

Comment:
For the sake of "whoa, such a very long line", I think we can dispense with the nameofs.

And be consistent with one or two equals signs.  Better yet, colons.

```suggestion
                Debug.Fail($"Caller is responsible for temporary decryption buffer creation destination. destination.Length: {destination.Length}, needed: {rsaSize}");
```